apache2/mod_security2.c

```diff
@@ -92,6 +92,11 @@ unsigned long int DSOLOCAL conn_write_state_limit = 0;
 TreeRoot DSOLOCAL *conn_write_state_whitelist = 0;
 TreeRoot DSOLOCAL *conn_write_state_suspicious_list = 0;
 
+#ifdef WAF_JSON_LOGGING_ENABLE
```

Comment:
I am not sure if we need this change. I looked at sample audit logs and there is already a ver variable that has the rule set type and version information (e.g. ver "OWASP_CRS/3.0.0"). So, somehow that information is already getting plumbed?

Reply:
yes, remove it.